observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,31 @@ These release notes track the development of the Network Observability Operator
+* Previously, the SR-IOV secondary interface was not detected if the interface was created first and then the eBPF agent was deployed. It was only detected if the agent was deployed first and then the SR-IOV interface was created.Now, the SR-IOV secondary interface is detected no matter the sequence of the deployments. (link:https://issues.redhat.com/browse/NETOBSERV-1697[*NETOBSERV-1697*])
+* Previously, when Loki was disabled, the *Topology* view in the OpenShift web console displayed the *Cluster* and *Zone* aggregation options in the slider beside the network topology diagram, even when the related features are not enabled. This has been fixed, the slider now only display options according to the enabled features. (link:https://issues.redhat.com/browse/NETOBSERV-1705[*NETOBSERV-1705*])
+* Previously, when Loki was disabled, and the OpenShift web console was first loading, an error would occur: `Request failed with status code 400 Loki is disabled`. With this fix, the errors no longer occur. (link:https://issues.redhat.com/browse/NETOBSERV-1706[*NETOBSERV-1706*])
+* Previously, in the *Topology* view of the OpenShift web console, when clicking on the *Step into* icon next to any graph node, the filters were not applied as required in order to set the focus to the selected graph node, resulting in showing a wide view of the *Topology*. Now the filters are correctly set, effectively narrowing down the *Topology*. As part of this change, clicking the *Step into* icon on a graph node now brings you to *Resource* scope instead of *Namespaces* scope. (link:https://issues.redhat.com/browse/NETOBSERV-1720[*NETOBSERV-1720*])
+* Previously, when Loki was disabled, in the *Topology* view of the OpenShift web console with the *Scope* set to *Owner*, clicking on the *Step into* icon next to any graph node would bring the *Scope* to *Resource*, which isn't available without Loki, so an error message was shown. Now the *Step into* icon is hidden in the *Owner* scope when Loki is disabled, so this scenario no longer occurs.(link:https://issues.redhat.com/browse/NETOBSERV-1721[*NETOBSERV-1721*])
+* Previously, when Loki was disabled, an error was displayed in the *Topology* view of the OpenShift web console when a group was set, but then the scope was changed so that the group becomes invalid. Now the invalid group is removed, preventing the error. (link:https://issues.redhat.com/browse/NETOBSERV-1722[*NETOBSERV-1722*])
+* When creating a `FlowCollector` resource from the OpenShift web console *Form view*, as opposed to the *YAML view*, the following settings are incorrectly managed by the web console: `agent.ebpf.metrics.enable` and `processor.subnetLabels.openShiftAutoDetect`. These settings can only be disabled in the *YAML view*, not in the *Form view*, . To avoid any confusion, these settings have been removed from the form view. They are still accessible in YAML. (link:https://issues.redhat.com/browse/NETOBSERV-1731[*NETOBSERV-1731*])
```

Comment:
```suggestion
* When creating a `FlowCollector` resource from the OpenShift web console *Form view*, as opposed to the *YAML view*, the following settings are incorrectly managed by the web console: `agent.ebpf.metrics.enable` and `processor.subnetLabels.openShiftAutoDetect`. These settings can only be disabled in the *YAML view*, not in the *Form view*. To avoid any confusion, these settings have been removed from the form view. They are still accessible in YAML. (link:https://issues.redhat.com/browse/NETOBSERV-1731[*NETOBSERV-1731*])
```

Reply:
Oof thanks 🥴